src/chainbase.cpp

```diff
@@ -24,22 +25,13 @@ namespace chainbase {
 
    void database::set_require_locking( bool enable_require_locking )
```

Comment:
This doesn't appear to be needed anymore. Is the reason to keep it to not make a breaking change to the interface of chainbase.hpp?

Since this is going into main which can be a breaking change (and I believe we already have breaking changes in main), I think it is perfectly fine to remove it. It doesn't do anything anyway so no one should be using it and I don't think there is any need for a deprecation first.

Reply:
It is not used in Leap. I was worried some users of Chainbase repo might use it. But you are right. The method is empty and does nothing. I am going to remove it. Thanks.